test_conformance/api/test_mem_object_info.cpp

```diff
@@ -18,21 +18,24 @@
 #include "harness/testHarness.h"
 
 
-#define TEST_MEM_OBJECT_PARAM( mem, paramName, val, expected, name, type, cast )    \
-error = clGetMemObjectInfo( mem, paramName, sizeof( val ), &val, &size );   \
-test_error( error, "Unable to get mem object " name );  \
-if( val != expected )   \
-{   \
-log_error( "ERROR: Mem object " name " did not validate! (expected " type ", got " type " from %s:%d)\n",   \
-expected, (cast)val, __FILE__, __LINE__ );   \
-return -1;  \
-}   \
-if( size != sizeof( val ) ) \
-{   \
-log_error( "ERROR: Returned size of mem object " name " does not validate! (expected %d, got %d from %s:%d)\n", \
-(int)sizeof( val ), (int)size , __FILE__, __LINE__ );   \
-return -1;  \
-}
+#define TEST_MEM_OBJECT_PARAM(mem, paramName, val, expected, name, type, cast) \
+    error = clGetMemObjectInfo(mem, paramName, sizeof(val), &val, &size);      \
+    test_error(error, "Unable to get mem object " name);                       \
+    if (val != expected)                                                       \
+    {                                                                          \
+        log_error("ERROR: Mem object " name                                    \
+                  " did not validate! (expected " type ", got " type           \
+                  " from %s:%d)\n",                                            \
+                  (cast)expected, (cast)val, __FILE__, __LINE__);              \
```

Comment:
The only change here besides the formatting is the `(cast)` in front of `expected`.